linera-core/src/updater.rs

```diff
@@ -241,7 +241,50 @@ where
             .collect::<Result<Vec<_>, _>>()?;
             result = self
                 .node
-                .handle_certificate(certificate.clone(), values, delivery)
+                .handle_certificate(certificate.clone(), values, BTreeMap::new(), delivery)
+                .await;
+        } else if let Err(NodeError::BlobsNotFound(blob_ids)) = &result {
```

Comment:
Here we also need to handle the case where there are both missing blobs _and_ missing certificate values.
(It's probably rare, so I think it's fine to retry with the values and then if there are missing blobs make another attempt with both the values and the blobs.)